script.js

```diff
@@ -1,4 +1,218 @@
+
+// function to generate random number
+function random(max) {
```

Comment:
Please try to name functions starting with a verb for clarity that these are functions. For example, this one could be `generateRandomNum`.